TCAT/Controllers/RouteDetail+ContentViewController.swift

```diff
@@ -183,10 +183,11 @@ class RouteDetailContentViewController: UIViewController, GMSMapViewDelegate, CL
 
         // Live Tracking Network Timer
         liveTrackingNetworkTimer?.invalidate()
-        liveTrackingNetworkTimer = Timer.scheduledTimer(timeInterval: liveTrackingNetworkRefreshRate, target: self, selector: #selector(getBusLocations),
-                                            userInfo: nil, repeats: true)
-        liveTrackingNetworkTimer?.fire()
-
+        if directions.contains(where: { $0.type != .walk }) {
```

Comment:
Were we calling our backend for walking directions? Good move to stop the call here as opposed to further down the line.